File: sub-model/src/main/java/org/github/base/mapper/ISysJobMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysJobEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 定时任务调度表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-12-12
 */
@MyBatisMapper
public interface ISysJobMapper extends IMapper<SysJobEntity> {}

File: sub-model/src/main/java/org/github/base/mapper/ISysJobMapper.kt
package org.github.base.mapper

import org.github.base.IMapper
import org.github.base.entity.SysJobEntity
import org.github.mybatis.MyBatisMapper

/**
 * <p>
 * 定时任务调度表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@MyBatisMapper
interface ISysJobMapper: IMapper<SysJobEntity>